app/views/content/how-to-apply-for-teacher-training.md

```diff
@@ -15,5 +15,5 @@ layout: "layouts/category"
 content:
     - content/how-to-apply-for-teacher-training/questions
     - content/how-to-apply-for-teacher-training/why-teach
-image: "static/images/content/blog/notepad.jpg"
+image: "static/images/content/how-to-apply-for-teacher-training/notepad.jpg"
```

Comment:
Nothing wrong with this but might suggest a shorter name eg `"static/images/content/application/notepad.jpg"`?